.github/workflows/nix.yml

```diff
@@ -68,6 +68,8 @@ jobs:
       - name: check environment
         run: |
           echo ${PATH} | tr ':' '\n'
+          lsb_release -a || true
+          ${{ matrix.profile.cc }} --version
```

Comment:
how about also enforcing that the compiler is the expected version , something like this

```
${{ matrix.profile.cc }} --version | grep -F " ${{ matrix.profile.version}}."
```

Reply:
... or more through

```
${{ matrix.profile.cc }} --version | grep -E "${{ matrix.compiler}}.* ${{ matrix.profile.version}}\."
```

Reply:
Good suggestion, but I think just having it informational is sufficient.